doc/guides/beyond-GSG.rst

```diff
@@ -122,6 +122,21 @@ repo.
    repository, switch branches in it, or perform a ``git bisect`` inside of
    it.
 
+Keeping Zephyr updated
+======================
+
+To update the Zephyr project source code, you need to get the latest
+changes via ``git``. Afterwards, run ``west update`` as mentioned in
+the previous paragraph.
+
+.. code-block:: console
+
+   # replace zephyrproject with the path you gave west init
+   cd zephyrproject
+   cd zephyr
```

Comment:
```suggestion
   cd zephyrproject/zephyr
```

Reply:
I was actually considering this first, but then thought that doesn't apply to Windows users (backslash).

On the other hand, the "Building blinky" section below also uses only forward slash, so I guess it's not so much of a problem.